packages/patternfly-4/react-core/src/components/Alert/Alert.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: 'Alert'
 cssPrefix: 'pf-c-alert'
+typescript: true
 ---
 import { Alert, AlertActionLink, AlertActionCloseButton } from '@patternfly/react-core';
-import './alert.scss';
+import './examples/alert.scss';
```

Comment:
I would really like to see these stay in /examples

Reply:
Yeah, we can fix that in a separate PR using explicit frontmatter.